lib/JobLogs/JobLogs.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MultiColumnList } from '@folio/stripes/components';
+
+import Preloader from '../Preloader';
+import { withJobLogsSort } from './withJobLogsSort';
+import { jobExecutionPropTypes } from '../Jobs';
+import {
+  defaultColumnMapping,
+  defaultVisibleColumns,
+  defaultColumnWidths,
+} from './defaultJobLogsProps';
+
+const JobLogs = props => {
+  const {
+    formatter,
+    contentData,
+    hasLoaded,
+    sortDirection,
+    sortField,
+    onSort,
+    visibleColumns,
+    columnWidths,
+    columnMapping,
+  } = props;
+
+  if (!hasLoaded) {
```

Comment:
```suggestion
  if (!hasLoaded) return <Preloader />;
```
Should we start using this approach for consistency for early return? I remember we had couple of places with such.